.yarnrc.yml

```diff
@@ -0,0 +1,341 @@
+yarnPath: .yarn/releases/yarn-2.4.1.cjs
+enableGlobalCache: true
+nodeLinker: node-modules
+compressionLevel: 0
+
+packageExtensions:
+  xunit-viewer@5.2.0:
+    peerDependencies:
+      "@babel/core": "*"
+
+  typeson-registry@1.0.0-alpha.35:
+    peerDependencies:
+      regenerator-runtime: "*"
+      core-js-bundle: "*"
+
+  realistic-structured-clone@2.0.2:
+    peerDependencies:
+      regenerator-runtime: "*"
+      core-js-bundle: "*"
+
+  reakit-warning@0.4.0:
+    peerDependencies:
+      react-dom: "*"
+
+  reakit-warning@0.5.5:
+    peerDependencies:
+      react-dom: "*"
+
+  reakit-warning@0.6.1:
+    peerDependencies:
+      react-dom: "*"
+
+  react-with-styles@3.2.3:
+    peerDependencies:
+      react-dom: "*"
+
+  react-live@1.12.0:
+    peerDependencies:
+      react: "*"
+
+  react-dev-utils@10.2.1:
+    peerDependencies:
+      webpack: "*"
+      typescript: "*"
+
+  jest-enzyme@7.1.2:
+    peerDependencies:
+      react: "*"
+
+  fake-indexeddb@3.1.0:
+    peerDependencies:
+      regenerator-runtime: "*"
+      core-js-bundle: "*"
+
+  "@emotion/native@10.0.27":
+    peerDependencies:
+      "@emotion/core": "*"
+      "react": "*"
+
+  "@storybook/core@6.1.10":
+    peerDependencies:
+      "typescript": "*"
+
+  "@storybook/react@6.1.10":
+    peerDependencies:
+      "typescript": "*"
+
+  "@types/wordpress__block-editor@2.2.9":
+    peerDependencies:
+      "react": "*"
+      "react-dom": "*"
+
+  "@types/wordpress__components@9.8.6":
+    peerDependencies:
+      "react": "*"
+
+  "@wordpress/block-editor@5.3.3":
+    peerDependencies:
+      "@wp-g2/create-styles": "*"
+      "react": "*"
+      "react-dom": "*"
+      "reakit-utils": "*"
+      "react-native": "*"
```

Comment:
Most of these are dependencies of `@wordpress/block-editor` that declare `peerDependencies`, but the `peerDependencies` aren't reproduced at `@wordpress/block-editor`. Is that accurate?

If that's the case, we can probably address these at the source and add them directly in Gutenberg.

Reply:
Correct. Yarn is very clear about transitive peer dependencies: if packageA consumes packageB, it must either provide all peerDependencies declared by packageB, or re-declare them as peerDependencies for packageA. Many packages, including `@wordpress/*` don't follow this.

The newest version of `@wordpress/*` doesn't use these dependencies anymore, but it wasn't released yet the last time I chedked.